testCourse/questions/demo/demoNewton-page1/question.html

```diff
@@ -0,0 +1,24 @@
+<pl-question-panel>
+<p>
+  Newton's laws of motion are three physical laws that, together, laid the foundation for classical mechanics.
+  They describe the relationship between a body and the forces acting upon it, and its motion in response to those forces.
```

Comment:
Same as other question (or even just remove the text since this is going to be in the test course only)